src/validation.cpp

```diff
@@ -2867,7 +2870,15 @@ static bool CheckBlockHeader(const CBlockHeader& block, CValidationState& state,
     if(!proofSize)
         return state.Error("No proof in block");
 
-    const CPubKey& aggregatePubkey = FederationParams().GetAggregatePubkey();
+    int height = 0;
+    if(chainActive.Tip())
+        height = chainActive.Tip()->nHeight;
+
+     CPubKey aggregatePubkey;
+    if(nHeight >= height)
```

Comment:
u mean remove the if else and only use 
FederationParams().GetAggPubkeyFromHeight(nHeight); ?
when create block coinbase it will reach the else and take
FederationParams().GetLatestAggregatePubkey();

Reply:
you mean height 0 represents new block? then we can leave it as it is.

Reply:
yes ok